PeerioTechnologies/peerio-mobile
# chore: removed error tracking setting [branch ch14375]

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/14375/mobile-include-usage-statistics-in-error-reporting-setting
#### Testing instructions  
Should only have one toggle button - "Share anonymous usage data" (this button was unchanged). The other toggle button, "Share crash & error reports", has been removed.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
